### PR TITLE
Import table filter

### PR DESCRIPTION
**Pull request by sstubbs**
### What problem does this PR solve? 
Sorry the incredibly late pr for this it fell off my radar. This should fix this issue https://github.com/pingcap/tidb-operator/issues/3183

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
```yaml
apiVersion: pingcap.com/v1alpha1
kind: Restore
metadata:
  name: production-0-restore-minio
spec:
  tableFilter:
    - "testing1.*"
#    - "testdb2.*"
  to:
    host: production-0-tidb
    port: 4000
    user: root
    secretName: backup-user-secret
  s3:
    provider: minio
    secretName: backup-object-store-secret
    endpoint: https://endpoint
    path: path/backup-2020-11-24T23:36:19Z.tgz
  serviceAccount: tidb-backup-manager
  storageSize: 10Gi
```

Code changes

 - Has Go code change

Side effects

 - There shouldn't be side effects with this

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
Support tableFilters for restoring with tidb-lightning
```

### CODE REVIEW

File: cmd/backup-manager/app/import/import.go [hunk 1]
```diff
@@ -97,6 +97,13 @@ func (ro *Options) loadTidbClusterData(restorePath string) error {
 		fmt.Sprintf("--d=%s", restorePath),
 		fmt.Sprintf("--tidb-port=%d", ro.Port),
 	}
+
+	if len(tableFilter) != 0 {
+		for _, tableFilter := range tableFilter {
+			args = append(args, "-f", tableFilter)
+		}
+	}
```
Comment (DanielZhangQD) [inline cmd/backup-manager/app/import/import.go:105]:
```suggestion
	for _, filter := range tableFilter {
		args = append(args, "-f", filter)
	}
```

Comment (DanielZhangQD) [reply cmd/backup-manager/app/import/import.go:105]: No need to check the length of a slice, right?
And to make the var name more clear.

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by lichunzhu — APPROVED** [review]
LGTM

**Pull request merged**